cmd/boskos/boskos.go

```diff
@@ -144,3 +172,69 @@ func main() {
 	// signal to the world that we're ready
 	health.ServeReady()
 }
+
+type configSyncReconciler struct {
```

Comment:
would it make sense to move the rest of this out of `cmd/boskos/boskos.go`?

Reply:
where to, a new package or just a different file?

Reply:
I was thinking "new package" but now I'm not sure it's really that necessary.